draft-ietf-ppm-dap.md

```diff
@@ -401,12 +405,6 @@ schemes that implement the VDAF interface specified in
 * Prio3 ({{Section 7 of !VDAF}}), which allows for aggregate statistics such as
   sum, mean, histograms, etc.
 
-* Poplar1 ({{Section 8 of !VDAF}}), which allows for finding the most popular
```

Comment:
I dropped references to Poplar1 throughout; I'm not sure if there is anything reasonable to replace it with. (Both PINE & Mastic, IIUC, are not yet to the point where we might cite them.)

Reply:
We could keep the "counting the number of Clients that hold a given string" use case for Poplar1. This is basically a limited form of attributed based metrics (i.e., Prio3Count with attributes).